src/ngRoute/route.js

```diff
@@ -551,14 +551,29 @@ function $RouteProvider() {
       }
     }
 
+    function buildResolvePromises(resolve) {
+      var locals = angular.extend({}, resolve);
```

Comment:
I would call `locals` `resolvePromises` to keep it consistent with the name of the function.